src/server/data/learners/DouglasLubaway.json

```diff
@@ -5,12 +5,12 @@
 	"name": "Douglas Lubaway",
 	"story": "Adipisicing elit aliquip nulla officia. Tempor sint cillum veniam dolore consectetur duis ut aliqua id laborum. Eiusmod ex ex ad ea amet sit occaecat do consequat amet.",
 	"projects": [{
-			"title": "/Blastoise.jpg",
-			"link": "https://github.com/hhhhhaaaa/book-store"
```

Comment:
just double-checking that we do indeed want to remove `hhhhhaaaa/book-store` and _replace_ it with `hhhhhaaaa/mutably-starter` ... yes?